Topics/starting-a-data-project.md

```diff
@@ -0,0 +1,254 @@
+
+Following the example above, this paper -> [A Survey of the Recent Architectures of Deep Convolutional Neural Networks](https://arxiv.org/abs/1901.06032) contains a detailed map of the most important papers on convolutional networks (neural networks that work well with images and videos) and their evolutions, up to the most advanced architectures. 
+
+Now you just have to look for the most important (or interesting) papers mentioned, organized is a time-aware tree!
+
+A good practice is to use [Zotero](https://www.zotero.org/), a document manager that allows you to keep track of all your research.
+
+You can then repeat this process in a more specific way, for example by looking for a survey paper on convolutional networks applied to the diagnosis of medical images. 
+
+#### Warning
+Before reading any paper [**read this!**](https://web.stanford.edu/class/ee384m/Handouts/HowtoReadPaper.pdf)
+
+It's a paper that explains how to read a paper. Yes, we at Virgilio like recursion.
+
+### Choosing a Project Structure
+Choosing a project structure is vital to managing the complexities that result from the evolution of the project. Without a clear structure, you'll find yourself with randomly scattered files, dataset versions with similar names, so much so that it hurts your head! 
+
+Well organized code tends to be self-documenting in that the organization itself provides context for your code without much overhead. People will thank you for this because they can:
+
+- Collaborate more easily with you on this analysis
+- Learn from your analysis about the process and the domain
+- Feel confident in the conclusions at which the project arrives
+
+
+But the first person to thank the ordered project structure is you! When we look at the code we wrote months ago, we often don't remember anything! 
+
+> "Mmmm... I don't remember if the good file was analysis.py, analysis_final.py, analysis_1.py" :-)
+
+For these reasons, good people have developed a fantastic project, [**Cookiecutter**](https://drivendata.github.io/cookiecutter-data-science/), which wants to standardize the structure of projects by providing a sensible and flexible template. 
+
+To create the project skeleton just install the package:
+```
+pip install cookiecutter
+```
+
+and then use:
+
+```
+cookiecutter https://github.com/drivendata/cookiecutter-data-science
+```
+
+You can customize the template according to your needs, just clone the repo, modify it and then use:
+
+```
+cookiecutter https://github.com/...... your repo .....
+```
+
+Cookiecutter projects have the following structure:
+
+
+![Figure 1-1](cookiecutter.png "1") 
+
+[Here](https://drivendata.github.io/cookiecutter-data-science/) you can find how to use it and the motivations behind the structure choices, and [here](https://cookiecutter.readthedocs.io/en/latest/readme.html) you can find the docs.
+
```

Comment:
I have a page full of useful python/datascience/ml related resources:
- https://github.com/neomatrix369/awesome-ai-ml-dl/blob/master/Programming-in-Python.md#cookie-cutter-python-project-templates
- https://github.com/neomatrix369/awesome-ai-ml-dl/blob/master/Programming-in-Python.md

See if you can add this somewhere in this document!

Reply:
added